### PR TITLE
add skonto to release roster & sign him up for v1.11 release

### PR DESCRIPTION
**Pull request by dprotaso**

### CODE REVIEW

**Comment by dprotaso** [comment]
/assign @skonto 

cc @pierDipi you'll want to find someone to help with eventing for v1.11 in July

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ The Knative project releases quarterly, on Tuesday of the 4th week of January, A
 | ------- | ------------ | -------------- | --------------------| ---------- | -----------
 | v1.9    | 2023-01-24   | psschwei       | evankanderson       | 2023-01-17 | 2023-01-25
 | v1.10   | 2023-04-25   | pradnyavmw, kvmware | dsimansk, pierDipi | 2023-04-18 | 2023-04-26
-| v1.11   | 2023-07-25   | TBD            | TBD                 | 2023-07-18 | 2023-07-26
+| v1.11   | 2023-07-25   | skonto         | TBD                 | 2023-07-18 | 2023-07-26
```
[the remark this answers is not in the document]
Comment (pierDipi) [reply README.md:22]: cc @creydr

[the remark this answers is not in the document]
Comment (matzew) [reply README.md:22]: For 1.12 I can do it again - but 1.11 is during summer vacation times

**Comment by dprotaso** [comment]
ok done - this just needs a LGTM

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

File: README.md [hunk 2]
```diff
@@ -15,12 +15,12 @@ The Knative project releases quarterly, on Tuesday of the 4th week of January, A
...
 ### Release schedule for 2023
 
-| Release | Release Date | Serving        | Eventing            | PKG cut    | Unpin repos
-| ------- | ------------ | -------------- | --------------------| ---------- | -----------
-| v1.9    | 2023-01-24   | psschwei       | evankanderson       | 2023-01-17 | 2023-01-25
-| v1.10   | 2023-04-25   | pradnyavmw, kvmware | dsimansk, pierDipi | 2023-04-18 | 2023-04-26
-| v1.11   | 2023-07-25   | TBD            | TBD                 | 2023-07-18 | 2023-07-26
-| v1.12   | 2023-10-24   | TBD            | TBD                 | 2023-10-17 | 2023-10-25
+| Release | Release Date | Serving             | Eventing            | PKG cut    | Unpin repos
+| ------- | ------------ | ------------------- | --------------------| ---------- | -----------
```
Comment (skonto) [inline README.md:19]: [](url)

**Pull request merged**